news-bundle/src/Resources/contao/modules/ModuleNews.php

```diff
@@ -147,6 +147,7 @@ protected function parseArticle($objArticle, $blnAddArchive=false, $strClass='',
 		$objTemplate->commentCount = $arrMeta['comments'] ?? null;
 		$objTemplate->timestamp = $objArticle->date;
 		$objTemplate->author = $arrMeta['author'] ?? null;
+		$objTemplate->authorModel = $arrMeta['authorModel'] ?? null;
```

Comment:
Would it not make more sense to add `$arrMeta` to the template? Then the users could access any metadata including the author model.

Reply:
So `$objTemplate->meta = $arrMeta;` you mean?

Reply:
Yes.